metron-deployment/development/centos6/README.md

```diff
@@ -103,3 +103,8 @@ Tags are listed in the playbooks, some frequently used tags:
 + `hdp-deploy` - Deploy and Start HDP Services (will start all Hadoop Services)
 + `sensors` - Deploy and start the sensors.
 + `sensor-stubs` - Deploy and start the sensor stubs.
+
+
+### Making Metron more stable for developers
```

Comment:
@sardell Thanks for the contribution! This is good info. Rather than duplicating this between centos and ubuntu, could we put it here
- https://github.com/sardell/metron/tree/8034d69e9d3cb7f807881011f6ad50f145f89be8/metron-deployment#how-do-i-deploy-metron-on-a-single-vm
or here?
- https://github.com/sardell/metron/blob/8034d69e9d3cb7f807881011f6ad50f145f89be8/metron-deployment/development/README.md

I'll leave it to your discretion.

Reply:
I would tend to agree with this and have this advice at the https://github.com/sardell/metron/blob/8034d69e9d3cb7f807881011f6ad50f145f89be8/metron-deployment/development/README.md level since the single node stuff people (probably misguidedly) use for purposes other than dev

Reply:
@simonellistonball @mmiklavc I'll go ahead and move this documentation to the metron-deployment/development level.